lib/opensusebasetest.pm

```diff
@@ -360,7 +360,12 @@ sub wait_boot {
     unlock_if_encrypted if !get_var('S390_ZKVM');
 
     if ($textmode || check_var('DESKTOP', 'textmode')) {
-        assert_screen [qw(linux-login emergency-shell emergency-mode)], $ready_time;
+        if (!check_screen [qw(linux-login emergency-shell emergency-mode)], $ready_time) {
+            #We are not able to boot due to bsc#980337
+            record_soft_failure 'bsc#980337';
```

Comment:
I've added check for TEST variable value. We also can add parameter which we will pass from user_defined_snapshot.